geoportailv3/static/js/maincontroller.js

```diff
@@ -109,6 +111,7 @@ app.MainController = function($scope, gettextCatalog, langUrls,
   this.setMap_();
   this.switchLanguage('fr');
   this.manageSelectedLayers_($scope, ngeoSyncArrays);
+  appExclusionManager.init(this['map']);
```

Comment:
We should add a private `this.map_` property instead of using/repeating `this['map']` in every controller method.